resources/github-actions.md

```diff
@@ -37,6 +37,89 @@ Additionally, the section [here](https://docs.gha-runners.nvidia.com/pull-reques
+| --------------------------------- | ------------------------------- | :--------------: | :-------------: | :-------------------------------------: |
+| - PRs                             | - `pr.yaml`                     | ✅               | ✅              | ❌                                      |
+| - `branch-*` Merges<br>- Releases | - `build.yaml`                  | ✅               | ❌              | ✅                                      |
+| - Nightlies                       | - `build.yaml`<br>- `test.yaml` | ✅               | ✅              | ✅                                      |
+
+Although release workflows don't run tests, they do go through a week of nightly testing to ensure everything works as expected. See [this page]({% link releases/process.md %}) for more details about the release process.
+
+## How Nightlies Are Triggered
+
+Since RAPIDS consists of a collection of libraries that depend on each other, it's important that nightly builds and tests run in the correct order.
+
+The [rapidsai/workflows](https://github.com/rapidsai/workflows) repository has a [nightly pipeline job](https://github.com/rapidsai/workflows/actions/workflows/nightly-pipeline.yaml) that is responsible for triggering jobs in the correct order.
+
+An example workflow run can be seen in the screenshot below.
+
+![](/assets/images/nightly_pipeline.png)
+
+## Subscribing to Nightlies
+
+A [recent blog post by GitHub](https://github.blog/changelog/2022-12-06-github-actions-workflow-notifications-in-slack-and-microsoft-teams/) explains how workflows can be subscribed to via Slack.
+
+The gist of the article is that the following command can be run in any Slack channel to subscribe that channel to a particular workflow:
+
+```sh
+/github subscribe owner/repo workflows:{name: "workflow_name"}
+```
+
+Multiple workflow names can also be passed to the command in order to subscribe to multiple workflows (shown in example below).
+
+For RAPIDS libraries, it is recommended to use the following commands to subscribe a particular Slack channel to branch build, nightly build, and nightly test workflow runs:
+
+```sh
+/github subscribe rapidsai/<repo> workflows:{name: "test","build"}
+/github unsubscribe rapidsai/<repo> issues pulls commits releases deployments
+```
+
+The second step is necessary because the `/github subscribe`` command will also subscribe the channel to a lot of other GitHub events, which will contribute a lot of noise.
+
+The `name` field in the `workflows` object corresponds to the name of a particular workflow (e.g. [this field](https://github.com/rapidsai/cudf/blob/66b846a01d418c51fb01126218d6b5e3fd83d405/.github/workflows/test.yaml#L1)).
+
+To only subscribe to nightly builds and nightly tests (and not branch builds), the actor filter can be used:
+
+```sh
+/github subscribe rapidsai/<repo> workflows:{name: "test","build", actor:"GPUtester"}
```

Comment:
Can we verify this? The syntax for `{name: "test","build", actor:"GPUtester"}` looks funny to me. Do the `name` entries need to be in an array?